api/gateway.proto

```diff
@@ -291,17 +291,22 @@ message GatewayConnectionStats {
   repeated SubBand sub_bands = 10;
 }
 
-// The plain text value of the gateway secret.
-message GatewaySecretPlainText{
-  map<string,bytes> values = 1 [(validate.rules).map.keys.string = {pattern: "^[a-z0-9](?:[-]?[a-z0-9]){2,}$" , max_len: 36}];
+// The plain text value of the Basic Station LNS gateway secret.
+message GatewayLBSLNSSecretPlainText{
+  string token_key = 1;
+}
+
+// The collective secret message use in RPC calls.
+message GatewaySecrets{
+  GatewayLBSLNSSecretPlainText lbs_lns_plain_text = 1 [(gogoproto.customname) = "LBSLNSPlainText"];
 }
```

Comment:
- `GatewaySecrets` would imply it is already plain text. Maybe clarify by renaming `GatewaySecretValue`
- Either make the secret value fields a `oneof`, or don't declare `GatewaySecrets` at all, and return a `bytes` to the client, and let the client do the unmarshaling to the expected type. That is not type-safe, but gives more flexiblity to the client, in the sense that it could also store JSON buffers

Reply:
Our API Keys are 98 characters so I've capped this at 128 characters.

Reply:
Either option would mean that we can only send one secret per call. That's fine right? 
Actually, each secret should be handled by a separate key so this is fine indeed.